### ISSUE TITLE
You aren't appreciated enough

### ISSUE THREAD

**Issue by BioMyth** [CelesteBlue-dev/PSVita-StorageMgr#32]
I see you getting way too much flak and not enough props for all the great work you've done(such as this plugin!) So Thanks!!! You're a rock star of the vita Homebrew scene!

**Comment by dmaskell92** [comment]
I agree, you've made my Vita better!

**Comment by dysprosi** [comment]
I second, er.. fourth? the motion! Thank you for all your hard work and continued updates and tweaks.

**Comment by shedeus** [comment]
Thanks for your tireless efforts.  storagemgr for life! :P

**Comment by burtman316** [comment]
Yes thanks just remember progress never comes with no mistakes and if nothing else you're making me learn linux ahhhhh!
